lib/rules/attribute-indentation.js

```diff
@@ -243,6 +243,7 @@ module.exports = class AttributeIndentation extends Rule {
 
       this.log({
         message,
+        node,
         line: actual.line,
         column: actual.column,
```

Comment:
Some rules add `node` and remove `line`/`column`/`source`, but others don't. Any particular reason?

Reply:
Some rules override the values of `line`, `column` and `source`, which end up being distinctly different than those from `node`. In these cases, we don't remove those properties.

See https://github.com/ember-template-lint/ember-template-lint/pull/2073/files#r698866054, for example.

Reply:
I ran the following in the `log` call to check if any of the current rules are passing in the same line/column as their node:

```js
for (let prop of ['line', 'column']) {
  if (prop in result && result[prop] === result.node.loc['prop']) {
    throw new Error(`result.${prop} and result.node.loc.${prop} are equal`);
  }
}
```

No rules appear to pass the same values.

Reply:
Thanks for checking that!